tests/conformance_ruby/tags/standard_tag_test.rs

```diff
@@ -40,7 +39,7 @@ fn test_has_a_block_which_does_nothing() {
     assert_template_result!("", "{% comment %}{% blabla %}{% endcomment %}");
     assert_template_result!("", "{%comment%}{% endif %}{%endcomment%}");
     assert_template_result!("", "{% comment %}{% endwhatever %}{% endcomment %}");
-    assert_template_result!("", "{% comment %}{% raw %} {{%%%%}}  }} { {% endcomment %} {% comment {% endraw %} {% endcomment %}");
+    assert_template_result!("", "{% comment %}{% raw %} {{%%%%}}  }} { {% endcomment %}{% comment %}{% endraw %} {% endcomment %}");
```

Comment:
What is this test change buying us?  Is the problem the fact that this results in leftover whitespace?  If so, this is actually masking https://github.com/cobalt-org/liquid-rust/issues/244 and we should be keeping to Ruby's versions of the tests.

Reply:
I noticed the whitespace change in the diff but somehow missed the `%}`.

Yuck.  It appears comments are still parsed, just the result is discarded
- Tag: https://github.com/Shopify/liquid/blob/master/lib/liquid/tags/comment.rb
- Base class: https://github.com/Shopify/liquid/blob/master/lib/liquid/block.rb

So the `raw` is being processed and everything within the `raw` is being discarded.

The negative to this is you can't have invalid content in your `comment` without putting it in a `raw`.  The positive is at least it means you can comment out `raw`.

Reply:
That is odd... It means that after commenting out a piece of code, it will still waste time getting rendered for nothing.

Maybe in the future it would be worthwhile to create an alternative comment block which trully ignores what's inside and have a way for the user to choose between the two alternatives (based on the same system that would allow choosing jekyll or shopify includes). This way people who wished to have liquid behave the same as ruby's counterpart would be able to, and those who'd like to save time by not rendering unused blocks would also have a choice.

Reply:
There are also those tests:
```
    assert_template_result!("", "{%comment%}{%blabla%}{%endcomment%}");
    assert_template_result!("", "{% comment %}{% blabla %}{% endcomment %}");
    assert_template_result!("", "{%comment%}{% endif %}{%endcomment%}");
    assert_template_result!("", "{% comment %}{% endwhatever %}{% endcomment %}");
```
How should comment distinguish between undefined tags like `{%blabla%}` and raw blocks?

Reply:
> That is odd... It means that after commenting out a piece of code, it will still waste time getting rendered for nothing.

They get parsed but the render never gets called.  If you want to comment out a `raw` that contains `comment`, then this is the only way.

> Maybe in the future it would be worthwhile to create an alternative comment block which trully ignores what's inside

This might get confusing when the workaround is just `{% comment %}{% raw %} ... whatever you want {% endraw %}{% comment %}` and is only saving you from having to spend time parsing the stuff in the `raw`.

Reply:
> There are also those tests:

Looking at the [ruby code](https://github.com/Shopify/liquid/blob/master/lib/liquid/block.rb#L66) my guess is these are subject to the regular unknown tag handling.  By default, the Ruby implementation has lax handling (ignored) but I assume if you turn on strict handling (the only mode liquid-rust supports), you'd get errors.

So my guess is we should expect these to fail.  In the past, I've either modified the test to pass or changed the test to a failing test.  In this case, I think it is intentionally covering an unknown tag.  So if unknown tags in comments should be a parse error, then I feel we should turn these into assertions that parsing failed.

If we aren't confident enough on this, could always open a "Question" issue and document the test with it.